passkit_ui/lib/passkit_ui.dart

```diff
@@ -1 +1,8 @@
-export 'src/pk_pass_widget.dart';
+export 'src/widgets/widgets.dart';
+export 'src/extensions/extensions.dart';
+export 'src/theme/theme.dart';
+export 'src/boarding_pass.dart';
+export 'src/coupon.dart';
+export 'src/event_ticket.dart';
+export 'src/generic.dart';
+export 'src/store_card.dart';
```

Comment:
I would, at least for now, keep the `PkPassWidget` as the only exposed class. That makes iterating a bit easier, while ensuring that no consumer of the library gets broken due to API changes.

In particular, the `PkPassTheme` is something that's not yet ready to be public facing, since it probably should be a `ThemeExtension` instead.